src/main/java/com/rtbhouse/utils/avro/FastDeserializerGenerator.java

```diff
@@ -55,6 +63,7 @@
 
         try {
             deserializerClass = classPackage._class(className);
+            deserializerClass.annotate(SuppressWarnings.class).param("value", "unchecked");
```

Comment:
What exactly caused suppression to be added? Maybe it can be avoided? We prefer to generate clean code.

Reply:
As you found out, it was caused by raw types usage - now that issue is solved.